src/components/auth/SignInScreen.js

```diff
@@ -78,85 +79,101 @@ export default class SignInScreen extends React.Component {
+                  placeholderTextColor={COLORS.forestgreen}
+                  returnKeyType='next'
+                  autoCapitalize='none'
+                  autoCorrect={false}
+                  keyboardType={'email-address'}
+                  onSubmitEditing={(event) => { this.refs.SecondInput._root.focus() }}
+                  onChangeText={value => this.onChangeText('email', value)}
+                />
+              </Item>
+              {/* Password */}
+              <Item style={styles.itemStyle}>
+                <Input
+                  style={styles.input}
+                  placeholder={"Senha"}
+                  placeholderTextColor={COLORS.forestgreen}
+                  returnKeyType='go'
+                  autoCapitalize='none'
+                  autoCorrect={false}
+                  secureTextEntry={true}
+                  ref='SecondInput'
+                  onChangeText={value => this.onChangeText('password', value)}
+                />
+              </Item>
+              {/* Sign Up Text */}
+              <Text style={styles.buttonText2}>Novo usuário?{" "}
+                <Text
+                  onPress={() => this.props.navigation.navigate('SignUp')}>
+                  <Text
+                    style={styles.buttonText2link}>Crie uma conta{" "}
                   </Text>
-                  {/* Loading ActivityIndicator */}
-                </View>  
-                <KeyboardAvoidingView style={styles.container2} behavior='position' enabled>
-                  {/* Sign In Button */}
-                  <TouchableOpacity
-                    onPress={()=>this.handleSignIn()}
-                    disabled={this.state.isLoading}
-                    style={styles.buttonStyle1}>
-                  {!this.state.isLoading &&
-                    <Text style={styles.buttonText1}>
-                      Entrar
+                  <MaterialCommunityIcons
+                    color={COLORS.forestgreen}
+                    name='chevron-double-right'
+                  />
+                </Text>
+                {" "}
+              </Text>
+              {/* Forgot Password Text */}
+              <Text style={styles.buttonText2}>Esqueceu sua senha?{" "}
+                <Text
+                  activeOpacity={0.8}
+                  onPress={() => this.props.navigation.navigate('ForgotPassword')}>
+                  <Text
+                    style={styles.buttonText2link}>Recupere aqui{" "}</Text>
+                  <MaterialCommunityIcons
+                    name='chevron-double-right'
+                    color={COLORS.forestgreen}
+                  />
+                </Text>
+                {" "}
+              </Text>
+              {/* Loading ActivityIndicator */}
+            </View>
+            <KeyboardAvoidingView
+              keyboardVerticalOffset={offset}
+              style={styles.container2}
+              behavior={Platform.OS == "ios" ? "position" : null}
+              enabled>
```

Comment:
![image](https://user-images.githubusercontent.com/5898380/87611440-eec10580-c6bc-11ea-96da-a2f3cf6ab0ff.png)
still not fixed, id say go back to what you had before, but try to reduce the gap between the button and the textinputs